buildSrc/subprojects/uber-plugins/src/main/kotlin/gradlebuild/java-library.gradle.kts

```diff
@@ -29,10 +29,11 @@ plugins {
     id("gradlebuild.classycle")
     id("gradlebuild.integration-tests")
     id("gradlebuild.cross-version-tests")
+    id("gradlebuild.ci-lifecycle")
```

Comment:
Not sure if it is a good idea to add a separate plugin here and not register the lifecycle tasks directly in java-library plugin.

Reply:
I think both solutions are fine.